### ISSUE TITLE
Strange Path

### ISSUE THREAD

**Issue by gpires** [PokemonGoF/PokemonGo-Bot#1173]
```

[11:25:13] Need to move closer to Pokestop
[=                                       ] 4%[11:25:25] Found fort d7924e8a0d7149da99ee3601dd267b44.16 at distance 0.02km
[11:25:25] Need to move closer to Pokestop
[=                                       ] 3%[11:25:37] Found fort d7924e8a0d7149da99ee3601dd267b44.16 at distance 0.02km
[11:25:37] Need to move closer to Pokestop
[                                        ] 1%[11:25:49] Found fort d7924e8a0d7149da99ee3601dd267b44.16 at distance 0.02km
[11:25:49] Need to move closer to Pokestop
[                                        ] 1%[11:26:00] Found fort d7924e8a0d7149da99ee3601dd267b44.16 at distance 0.02km
[11:26:00] Need to move closer to Pokestop
[=                                       ] 3%[11:26:12] Found fort d7924e8a0d7149da99ee3601dd267b44.16 at distance 0.02km
[11:26:12] Need to move closer to Pokestop
[=                                       ] 3%[11:26:24] Found fort d7924e8a0d7149da99ee3601dd267b44.16 at distance 0.02km
[11:26:24] Need to move closer to Pokestop
[                                        ] 2%[11:26:37] Found fort d7924e8a0d7149da99ee3601dd267b44.16 at distance 0.02km
[11:26:37] Need to move closer to Pokestop
[=                                       ] 3%[11:26:43]
```
http://i.imgur.com/yAgRL09.png

### Other Information
OS: Win10
Git Commit: 7024c14433828c476ea04213afa82b6f2cdbc39d

**Comment by knobunc** [comment]
I had a similar issue starting with "2 commonwealth ave, boston ma"

**Comment by anonymouslulz22** [comment]
same issue

**Comment by binarydepartment** [comment]
![Strange Path](http://pix.toile-libre.org/upload/original/1469616456.png)

**Comment by knobunc** [comment]
Not sure if this helps, but mine is saying:

> [06:58:14] Need to move closer to Pokestop
> [====                                    ] 11%[06:58:22] Found fort 7418864b10cc4d7589f475270003e48a.16 at distance 0.06km
> [06:58:22] Need to move closer to Pokestop
> [====                                    ] 12%[06:58:30] Found fort 7418864b10cc4d7589f475270003e48a.16 at distance 0.07km
> [06:58:30] Need to move closer to Pokestop
> [=====                                   ] 13%[06:58:38] Found fort 7418864b10cc4d7589f475270003e48a.16 at distance 0.07km
> [06:58:38] Need to move closer to Pokestop
> [=====                                   ] 13%[06:58:45] Found fort 7418864b10cc4d7589f475270003e48a.16 at distance 0.06km
> [06:58:45] Need to move closer to Pokestop
> [====                                    ] 12%[06:58:53] Found fort 7418864b10cc4d7589f475270003e48a.16 at distance 0.07km
> [06:58:53] Need to move closer to Pokestop
> [=====                                   ] 13%[06:59:01] Found fort 7418864b10cc4d7589f475270003e48a.16 at distance 0.07km

And getting further away from the target, moving in zig-zags directly towards the target and away... resulting in a drunken walk about 90 degrees off from the direction it should be heading.

**Comment by robhunt3r** [comment]
Same here...

> [12:56:12] Need to move closer to Pokestop
[                                        ] 2%[12:56:20] Found fort 11959c3a72d948f4a3566cc6d0a80458.12 at distance 0.09km
[12:56:20] Need to move closer to Pokestop
[                                        ] 2%[12:56:30] Found fort 11959c3a72d948f4a3566cc6d0a80458.12 at distance 0.09km
[12:56:30] Need to move closer to Pokestop
[=                                       ] 3%[12:56:39] Found fort 11959c3a72d948f4a3566cc6d0a80458.12 at distance 0.09km
[12:56:39] Need to move closer to Pokestop
[                                        ] 2%[12:56:49] Found fort 11959c3a72d948f4a3566cc6d0a80458.12 at distance 0.09km
[12:56:49] Need to move closer to Pokestop
[=                                       ] 3%[12:56:58] Found fort 11959c3a72d948f4a3566cc6d0a80458.12 at distance 0.09km
[12:56:58] Need to move closer to Pokestop
[==                                      ] 5%[12:57:07] Found fort 11959c3a72d948f4a3566cc6d0a80458.12 at distance 0.10km
[12:57:07] Need to move closer to Pokestop
[==                                      ] 6%[12:57:16] Found fort 11959c3a72d948f4a3566cc6d0a80458.12 at distance 0.10km
[12:57:16] Need to move closer to Pokestop
[==                                      ] 7%[12:57:25] Found fort 11959c3a72d948f4a3566cc6d0a80458.12 at distance 0.10km
[12:57:25] Need to move closer to Pokestop
[==                                      ] 7%[12:57:33] Found fort 11959c3a72d948f4a3566cc6d0a80458.12 at distance 0.10km
[12:57:33] Need to move closer to Pokestop
[==                                      ] 6%[12:57:43] Found fort 11959c3a72d948f4a3566cc6d0a80458.12 at distance 0.10km
[12:57:43] Need to move closer to Pokestop
[===                                     ] 8%[12:57:52] Found fort 11959c3a72d948f4a3566cc6d0a80458.12 at distance 0.10km
[12:57:52] Need to move closer to Pokestop
[==                                      ] 7%[12:58:01] Found fort 11959c3a72d948f4a3566cc6d0a80458.12 at distance 0.10km
[12:58:01] Need to move closer to Pokestop
[==                                      ] 7%[12:58:10] Found fort 11959c3a72d948f4a3566cc6d0a80458.12 at distance 0.10km
[12:58:10] Need to move closer to Pokestop
[==                                      ] 7%[12:58:19] Found fort 11959c3a72d948f4a3566cc6d0a80458.12 at distance 0.10km
[12:58:19] Need to move closer to Pokestop
[==                                      ] 5%[12:58:28] Found fort 11959c3a72d948f4a3566cc6d0a80458.12 at distance 0.10km
[12:58:28] Need to move closer to Pokestop
[==                                      ] 7%[12:58:37] Found fort 11959c3a72d948f4a3566cc6d0a80458.12 at distance 0.10km
[12:58:37] Need to move closer to Pokestop
[==                                      ] 7%[12:58:47] Found fort 11959c3a72d948f4a3566cc6d0a80458.12 at distance 0.10km
[12:58:47] Need to move closer to Pokestop
[==                                      ] 6%[12:59:01] Found fort 11959c3a72d948f4a3566cc6d0a80458.12 at distance 0.10km
[12:59:01] Need to move closer to Pokestop
[===                                     ] 8%[12:59:18] Found fort 11959c3a72d948f4a3566cc6d0a80458.12 at distance 0.10km
[12:59:18] Need to move closer to Pokestop
[===                                     ] 9%[12:59:27] Found fort 11959c3a72d948f4a3566cc6d0a80458.12 at distance 0.10km
[12:59:27] Need to move closer to Pokestop
[====                                    ] 10%[12:59:37] Found fort 11959c3a72d948f4a3566cc6d0a80458.12 at distance 0.10km
[12:59:37] Need to move closer to Pokestop
[===                                     ] 9%[12:59:46] Found fort 11959c3a72d948f4a3566cc6d0a80458.12 at distance 0.11km
[12:59:46] Need to move closer to Pokestop
[====                                    ] 11%[12:59:55] Found fort 11959c3a72d948f4a3566cc6d0a80458.12 at distance 0.11km
[12:59:55] Need to move closer to Pokestop
[====                                    ] 10%[13:00:04] Found fort 11959c3a72d948f4a3566cc6d0a80458.12 at distance 0.11km
[13:00:04] Need to move closer to Pokestop
[====                                    ] 10%[13:00:13] Found fort 11959c3a72d948f4a3566cc6d0a80458.12 at distance 0.11km
[13:00:13] Need to move closer to Pokestop
[====                                    ] 11%[13:00:21] Found fort 11959c3a72d948f4a3566cc6d0a80458.12 at distance 0.11km
[13:00:21] Need to move closer to Pokestop
[====                                    ] 11%[13:00:30] Found fort 11959c3a72d948f4a3566cc6d0a80458.12 at distance 0.11km
[13:00:30] Need to move closer to Pokestop

**Comment by ayiire** [comment]
Dev release -18f232bffb8df487654e6c52ff3eea859a120925

Same thing for me.

[06:57:50] A Wild Doduo appeared! [CP 129] [Potential 0.62]
[06:57:50] IV [Stamina/Attack/Defense] = [9/15/4]
[06:57:55] Using Pokeball (chance: 62.29%)... (65 left!)
[06:57:55] Captured Doduo! [CP 129] [9/15/4]
[06:58:00] Found fort af5f82e1db264a4781ad58460a756212.16 at distance 0.36km
[06:58:00] Need to move closer to Pokestop
[                                        ] 0%[06:58:05] Scanning area for objects....
[06:58:05] Walking from (40.71902849450184, -73.88182811117511) to [40.720833000000006, -73.881877] 0.20km
[                                        ] 0%[06:58:14] Found fort af5f82e1db264a4781ad58460a756212.16 at distance 0.36km
[06:58:14] Need to move closer to Pokestop
[                                        ] 2%[06:58:24] Found fort af5f82e1db264a4781ad58460a756212.16 at distance 0.35km
[06:58:24] Need to move closer to Pokestop
[==                                      ] 5%[06:58:33] Found fort af5f82e1db264a4781ad58460a756212.16 at distance 0.35km
[06:58:33] Need to move closer to Pokestop
[====                                    ] 10%[06:58:43] Found fort af5f82e1db264a4781ad58460a756212.16 at distance 0.34km
[06:58:43] Need to move closer to Pokestop
[====                                    ] 12%[06:58:52] Found fort af5f82e1db264a4781ad58460a756212.16 at distance 0.34km
[06:58:52] Need to move closer to Pokestop

**Comment by robhunt3r** [comment]
Seems fixed by adding


```
self.api.player_update(latitude=lat, longitude=lng)
response_dict = self.api.call()
```

Above `logger.log('Arrived at Pokestop')` on move_to_fort_worker.py

**Comment by gpires** [comment]
@robhunt3r:  It remains the same

**Comment by robhunt3r** [comment]
@gpires yes, I was just editing my comment.. It seemed to be working but I'm stuck again, increasing distance between spots.... I will take another look...

**Comment by gpires** [comment]
There is a fix waiting to be merged :)
pull/1185

**Comment by gpires** [comment]
Fixed on 3a18aa57dfb67e2e09c207087cad6e5aa1867112
